### PR TITLE
WRK-1267 Fixed warnings

### PR DESCRIPTION
**Pull request by improbable-andreaskrugersen**
Fixed some warnings that only occur on /Wall

### CODE REVIEW

File: workers/c_client_direct/client.c [hunk 1]
```diff
@@ -114,7 +114,7 @@ int main(int argc, char** argv) {
   params.network.tcp.multiplex_level = 4;
   params.default_component_vtable = &default_vtable;
   Worker_ConnectionFuture* connection_future =
-      Worker_ConnectAsync(argv[1], atoi(argv[2]), argv[3], &params);
+      Worker_ConnectAsync(argv[1], (uint16_t)atoi(argv[2]), argv[3], &params);
```
Comment (dgavedissian) [inline workers/c_client_direct/client.c:117]: Should be `static_cast`

**Review by dgavedissian — APPROVED** [review]
LGTM. Sorry, missed this til now.

File: workers/c_client_direct/client.c (hunk 1 above)
Comment (improbable-andreaskrugersen) [reply workers/c_client_direct/client.c:117]: It's a c file :wink:

Comment (dgavedissian) [reply workers/c_client_direct/client.c:117]: haha, whoops.

**Pull request merged**